opentreemap/treemap/instance.py

```diff
@@ -112,6 +113,12 @@ def get_instance_permission_spec():
                              'of a tree or map feature'),
             'default_role_names': [Role.ADMINISTRATOR, Role.EDITOR],
             'label': _('Can View External Link')
+        },
+        {
+            'codename': PERMISSION_ACCESS_WORKS_MANAGEMENT,
+            'description': _('Can view or modify Works Management entities'),
+            'default_role_names': [Role.ADMINISTRATOR],
+            'label': _('Can Access Works Management')
```

Comment:
I would expect the Permission to appear on the Roles page for any instance that has Works Management enabled.

Since, for now, we're making Works Management universally enabled for all instances in the Development stack, it would follow that the Roles page should display it universally in that stack.

And since the default permission agreed on in the slack channel is `administrator` only, I'd expect to see it granted only to the Administrator role unless you change it.

What were you expecting, @RickMohr?

(Happy to change "access" to lower case).